libigcc/dot_commands.py

```diff
@@ -88,8 +106,11 @@ def dot_w( runner ):
 	".l" : ( "List the code you have entered", dot_l ),
 	".L" : ( "List the whole program as given to the compiler", dot_L ),
 	".n" : ( "Clear all entered commands ('new')", dot_n ),
+	".N" : ( "Clear all entered commands including user functions ('New')", dot_N ),
```

Comment:
I may change .n to take arguments instead, so you can choose which categories you want to wipe.